src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/function.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+    "bindings": [
+        {
+            "type": "httpTrigger",
+            "authLevel": "anonymous",
+            "name": "req",
+            "direction": "in",
+            "methods": [ "get", "post" ],
+            "route": "{x:regex(^(warmup|csharphttpwarmup)$)}"
```

Comment:
Currently Antares is using "csharphttpwarmup". I'd like to change that to "warmup" eventually, so we support both here.